packages/channels/botpress-channel-web/src/views/web/side/index.jsx

```diff
@@ -153,14 +156,14 @@ export default class Side extends React.Component {
     }
 
     return (
-      <span className={style.downloadIcon}>
+      <span className={`${style.downloadIcon} botpress__downloadIcon`}>
```

Comment:
Please keep the consistent case for classes, like if you're using `kebab-case` already keep using it, don't mix with `camelCase` here